api/tests/src/data_migration/transformation/subtask/test_transform_opportunity_attachment.py

```diff
@@ -1,41 +1,101 @@
-from src.db.models.foreign.attachment import TsynopsisAttachment as TsynopsisAttachmentF
-from src.db.models.staging.attachment import TsynopsisAttachment as TsynopsisAttachmentS
-from tests.src.db.models.factories import (
-    ForeignTsynopsisAttachmentFactory,
-    StagingTsynopsisAttachmentFactory,
+import pytest
+
+import tests.src.db.models.factories as f
+from src.data_migration.transformation.subtask.transform_opportunity_attachment import (
+    TransformOpportunityAttachment,
+)
+from tests.src.data_migration.transformation.conftest import (
+    BaseTransformTestClass,
+    setup_opportunity_attachment,
+    validate_opportunity_attachment,
 )
 
 
-def test_uploading_attachment_staging(db_session, enable_factory_create, tmp_path):
```

Comment:
Because these tests use the staging table factories heavily, we don't need the specific tests anymore